### PR TITLE
Moved old stuff over

### PR DESCRIPTION
**Pull request by MaybeWilli**
Hardware interface, moveit controller, moveit bridge node, etc

### CODE REVIEW

**Review by ChrisRusu1 — APPROVED** [review]
Just saw you compile LGTM

File: src/arm_interface/arm_interface/joystickArmController.py
Comment (ChrisRusu1) [inline src/arm_interface/arm_interface/joystickArmController.py:1]: Look at commit 7d0b23c To change this to compile on jetson vs laptop

**Review by ChrisRusu1 — CHANGES_REQUESTED** [review]
Look at the most recent changes I made to 2023Rover in the arm interface, commit 7d0b23c. I made it so that the gpio is able to turn on and off if its on the jetson to make it easier to move off rover for testing without compiling GPIO

**Review by ConnorNeed — CHANGES_REQUESTED** [review]
Let's close this and create a more targeted pull request

**Pull request closed without merging**